plugins/board-resources/src/components/editor/CardDetails.svelte

```diff
@@ -85,17 +52,12 @@
 </script>
 
 {#if value}
-  {#if members && members.length > 0}
+  {#if value.members?.length}
     <div class="flex-col mt-4 mr-6">
       <div class="text-md font-medium">
         <Label label={board.string.Members} />
       </div>
-      <div class="flex-row-center flex-gap-1">
-        {#each members as member}
-          <MemberPresenter value={member} size="large" menuItems={getMenuItems(member)} />
-        {/each}
-        <Button icon={IconAdd} shape="circle" kind="no-border" size="large" on:click={membersHandler} />
-      </div>
+      <MembersPresenter object={value} {membersHandler} />
```

Comment:
Can we revert these changes and use `UserBoxList` instead, like here https://github.com/hcengineering/anticrm/blob/main/plugins/board-resources/src/components/KanbanCard.svelte#L198

Reply:
Can I change `items` to `value` or it's better to use proxy?
![Screenshot from 2022-05-16 23-04-22](https://user-images.githubusercontent.com/41288429/168635930-3558877d-8500-4b6e-b6d2-b124a01aa2f7.png)